src/py/mat3ra/notebooks_utils/plot.py

```
@@ -1,8 +1,30 @@
+configure_matplotlib_renderer()
+
+__all__ = [
+    "display_matplotlib_figure",
+    "plot_distribution_function",
+    "plot_series",
+    "create_scatter_plot_2d",
+    "create_realtime_plot",
+    "create_update_callback",
+    "plot_2d_heatmap",
+    "plot_3d_surface",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)